### PR TITLE
Feature/use strict

### PR DESCRIPTION
**Pull request by tiinaromu**
#11

Did you mean this?

### CODE REVIEW

File: public/javascripts/filter.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+"use strict";
```
Comment (phadej) [inline public/javascripts/filter.js]: this doesn't have any effect in browser, as it isn't inside function.

File: public/javascripts/filter.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
 $(function() {
```
Comment (phadej) [inline public/javascripts/filter.js:1]: ... `"use strict"` should be here.

File: routes/index.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
-/* GET home page. */
+"use strict";
```
Comment (phadej) [inline routes/index.js:1]: OTOH, node.js files are ok this way. :+1:

**Comment by tiinaromu** [comment]
Like this?

**Comment by phadej** [comment]
exactly.

**Pull request merged**